python_etl/example_job.py

```diff
@@ -0,0 +1,39 @@
+from datetime import date, timedelta
```

Comment:
I think it would be better if every ETL job has its own folder, i.e. package, as some jobs might require multiple modules.